content/en/blog/2019/v1beta1-authorization-policy/index.md

```diff
@@ -0,0 +1,283 @@
+| Ingress/Egress gateway support | Not supported | Supported |
+
+Beyond all the differences, the `v1beta1` policy is enforced by the same engine in Envoy
+and supports the same authenticated identity (mutual TLS or JWT), condition and other
+primitives (e.g. IP, port and etc.) as the `v1alpha1` policy.
+
+See [authorization concept page](/docs/concepts/security/#authorization) for a detailed
+in-depth explanation of the `AuthorizationPolicy`.
+
+## Migration
+
+Istio 1.4 supports both `v1alpha` and `v1beta1` policies at the same time. The `v1beta1` policy
+overrides the `v1alpha1` policy if they are both applied to the same workload:
+
+1. If any `v1beta1` policies apply to the workload, the `v1beta1` policy is used and
+   the `v1alpha1` policies on the same workload will all be ignored
+1. If no `v1beta1` policy applies to the workload, the `v1alpha1` policy is used if
+   there is any
+
+{{< warning >}}
+This means when migrating to the `v1beta1`, you need to make sure the converted
+`v1beta1` policy covers all the `v1alpha1` policies applied to the same workload. The
+`v1alpha1` policies applied for the workload will be ignored after you applied the `v1beta1` policies.
+{{< /warning >}}
+
+It's recommended to migrate the `v1alpha1` policy workload-by-workload to make sure you don't
+forget any `v1alpha1` policies:
+
+1. For a given workload, find out all the `v1alpha1` policies applied to it
+1. Convert all the `v1alpha1` policies to `v1beta1` for the workload
+1. Apply the `v1beta1` policy and monitor the traffic to make sure the new policy is working as expected
+1. Continue the process for the next workload until all `v1alpha1` policies are converted
+1. Now you can remove all your `v1alpha1` policies
+
+To help ease the migration, Istio provides the command `istioctl x auth migrate` to automate the
+conversion from `v1alpha1` to `v1beta1`. The command fetches all the `v1alpha1` policies currently
+applied in the mesh and converts them to the `v1beta1` policy. The tool finds out the service-workload
+mapping by looking at the current services applied in the mesh.
+
+The command does the conversion at its best effort but may not be able to handle some
+corner cases. In this case, the tool will return an error and you will need to manually convert
+the policies.
+
+## Summary
+
+The Istio `v1beta1` authorization policy is a major update to the current `v1alpha1` RBAC policy
+with the design principle of aligning with Istio configuration model, improving user
+experience by simplifying the API and making policy semantics more flexible without
+introducing much complexity.
+
+The `v1beta1` policy is not backward compatible and requires a one time manual conversion. A tool
+is provided to automate this process.
+
+The previous configuration resources `ClusterRbacConfig`, `ServiceRole`, and `ServiceRoleBinding`
+will not be supported from Istio 1.6 onwards.
```

Comment:
This needs to be more explicit I think, not just a minor comment in the summary. Reading through this the first thing I was looking for was how long v1alpha1 is supported and I almost missed it

Reply:
Good point, I added a new section `Future of the `v1alpha1` RBAC policy` before the `Migration`.